### PR TITLE
Contactinfo spreadsheet

### PR DESCRIPTION
**Pull request by mgenova12**

### CODE REVIEW

**Comment by mgenova12** [comment]
https://trello.com/c/6BM7G4SO/126-as-an-admin-i-want-to-be-able-to-download-the-names-and-contact-information-of-applicants-registrants-in-spreadsheet-form

File: app/controllers/applications_controller.rb [hunk 1]
```diff
@@ -79,6 +79,29 @@ def completed_applications
     else 
       redirect_to root_path
     end
+
+  end
+
+  def download_spreadsheet
+
+    applications = Application.all.where(application_status: 'finished')
```
Comment (trevordjones) [inline app/controllers/applications_controller.rb]: Let's just use `Application.where(application_status: 'finished')`. Using `Application.all` will load all of the applications into memory and we don't need them all.

If you want to get ambitious, you can look at model `scopes` and see how they can be applied here. The goal would be to be able to write `Application.finished` - nice and clean.

File: app/views/applications/completed_applications.html.erb [hunk 2]
```diff
@@ -7,6 +7,8 @@
       <th>View Application</th>
     </tr>
 
+  <%= button_to "Download Contact Info", "/download_applicats", :method => :post, :class => 'btn btn-primary'%>
```
Comment (trevordjones) [inline app/views/applications/completed_applications.html.erb]: `download_applicants`

File: config/routes.rb [hunk 3]
```diff
@@ -14,6 +14,8 @@
   get "/calendar" => 'pages#calendar'
 
   get "/completed_applications" => 'applications#completed_applications'
+  post "/download_applicats" => 'applications#download_spreadsheet'
```
Comment (trevordjones) [inline config/routes.rb]: `download_applicants` - can we make it a `get` instead of a `post`?

**Pull request merged**